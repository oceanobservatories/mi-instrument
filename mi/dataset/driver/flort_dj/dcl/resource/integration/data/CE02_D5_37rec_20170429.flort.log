2017/04/29 00:00:12.105 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/29 23:45:59.356 04/29/17	23:44:46	700	386	695	184	460	67	554
2017/04/29 23:46:12.913 04/29/17	23:44:59	700	560	695	174	460	63	554
2017/04/29 23:46:19.690 04/29/17	23:45:06	700	446	695	184	460	69	554
2017/04/29 23:46:20.820 04/29/17	23:45:07	700	380	695	195	460	65	554
2017/04/29 23:46:24.208 04/29/17	23:45:11	700	504	695	178	460	64	554
2017/04/29 23:46:25.338 04/29/17	23:45:12	700	643	695	192	460	66	554
2017/04/29 23:46:26.468 04/29/17	23:45:13	700	481	695	198	460	64	554
2017/04/29 23:46:27.597 04/29/17	23:45:14	700	426	695	182	460	64	553
2017/04/29 23:46:38.895 04/29/17	23:45:25	700	392	695	184	460	64	553
2017/04/29 23:46:42.284 04/29/17	23:45:29	700	421	695	157	460	66	553
2017/04/29 23:46:43.414 04/29/17	23:45:30	700	403	695	178	460	63	553
2017/04/29 23:46:44.548 04/29/17	23:45:31	700	469	695	182	460	63	553
2017/04/29 23:46:51.321 04/29/17	23:45:38	700	348	695	187	460	65	553
2017/04/29 23:46:52.451 04/29/17	23:45:39	700	397	695	204	460	64	553
2017/04/29 23:46:53.581 04/29/17	23:45:40	700	482	695	168	460	64	553
2017/04/29 23:46:54.711 04/29/17	23:45:41	700	414	695	184	460	64	553
2017/04/29 23:46:55.840 04/29/17	23:45:42	700	573	695	198	460	66	553
2017/04/29 23:47:09.397 04/29/17	23:45:56	700	422	695	173	460	63	553
2017/04/29 23:47:10.527 04/29/17	23:45:57	700	496	695	181	460	67	553
2017/04/29 23:47:11.657 04/29/17	23:45:58	700	475	695	202	460	65	552
2017/04/29 23:47:12.787 04/29/17	23:45:59	700	471	695	185	460	64	552
2017/04/29 23:47:13.916 04/29/17	23:46:00	700	399	695	188	460	67	553
2017/04/29 23:47:15.046 04/29/17	23:46:02	700	497	695	203	460	66	552
2017/04/29 23:47:24.081 04/29/17	23:46:11	700	561	695	183	460	67	552
2017/04/29 23:47:25.211 04/29/17	23:46:12	700	411	695	206	460	65	552
2017/04/29 23:47:26.341 04/29/17	23:46:13	700	479	695	177	460	64	552
2017/04/29 23:47:31.991 04/29/17	23:46:18	700	493	695	182	460	64	552
2017/04/29 23:47:38.768 04/29/17	23:46:25	700	397	695	167	460	67	552
2017/04/29 23:47:42.203 04/29/17	23:46:29	700	483	695	182	460	66	552
2017/04/29 23:47:43.287 04/29/17	23:46:30	700	467	695	189	460	67	552
2017/04/29 23:47:44.417 04/29/17	23:46:31	700	390	695	195	460	65	552
2017/04/29 23:47:45.546 04/29/17	23:46:32	700	469	695	208	460	68	552
2017/04/29 23:47:46.676 04/29/17	23:46:33	700	409	695	203	460	65	552
2017/04/29 23:47:47.806 04/29/17	23:46:34	700	427	695	232	460	64	552
2017/04/29 23:47:54.584 04/29/17	23:46:41	700	437	695	231	460	63	552
2017/04/29 23:48:01.570 [flort:DLOGP2]:Instrument Stopped [Power Off]
