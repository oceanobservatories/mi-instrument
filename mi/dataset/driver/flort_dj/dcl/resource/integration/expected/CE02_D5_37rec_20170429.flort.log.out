
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498359.3559999466 |          5 |         3702498286 | 3702498359.3559999466 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             386 |              67 |            184 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498372.9130001068 |          5 |         3702498299 | 3702498372.9130001068 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             560 |              63 |            174 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498379.6900000572 |          5 |         3702498306 | 3702498379.6900000572 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             446 |              69 |            184 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498380.8199996948 |          5 |         3702498307 | 3702498380.8199996948 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             380 |              65 |            195 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498384.2080001831 |          5 |         3702498311 | 3702498384.2080001831 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             504 |              64 |            178 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498385.3380002975 |          5 |         3702498312 | 3702498385.3380002975 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             643 |              66 |            192 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498386.4679999352 |          5 |         3702498313 | 3702498386.4679999352 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             481 |              64 |            198 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498387.5970001221 |          5 |         3702498314 | 3702498387.5970001221 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             426 |              64 |            182 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498398.8949999809 |          5 |         3702498325 | 3702498398.8949999809 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             392 |              64 |            184 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498402.2839999199 |          5 |         3702498329 | 3702498402.2839999199 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             421 |              66 |            157 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498403.4140000343 |          5 |         3702498330 | 3702498403.4140000343 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             403 |              63 |            178 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498404.5480003357 |          5 |         3702498331 | 3702498404.5480003357 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             469 |              63 |            182 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498411.3210000992 |          5 |         3702498338 | 3702498411.3210000992 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             348 |              65 |            187 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498412.4510002136 |          5 |         3702498339 | 3702498412.4510002136 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             397 |              64 |            204 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498413.5810003281 |          5 |         3702498340 | 3702498413.5810003281 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             482 |              64 |            168 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498414.7109999657 |          5 |         3702498341 | 3702498414.7109999657 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             414 |              64 |            184 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498415.8400001526 |          5 |         3702498342 | 3702498415.8400001526 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             573 |              66 |            198 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498429.3970003128 |          5 |         3702498356 | 3702498429.3970003128 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             422 |              63 |            173 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498430.5269999504 |          5 |         3702498357 | 3702498430.5269999504 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             496 |              67 |            181 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498431.6570000648 |          5 |         3702498358 | 3702498431.6570000648 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             475 |              65 |            202 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498432.7869997025 |          5 |         3702498359 | 3702498432.7869997025 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             471 |              64 |            185 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498433.9159998894 |          5 |         3702498360 | 3702498433.9159998894 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             399 |              67 |            188 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498435.0460000038 |          5 |         3702498362 | 3702498435.0460000038 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             497 |              66 |            203 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498444.0810003281 |          5 |         3702498371 | 3702498444.0810003281 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             561 |              67 |            183 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498445.2109999657 |          5 |         3702498372 | 3702498445.2109999657 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             411 |              65 |            206 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498446.3410000801 |          5 |         3702498373 | 3702498446.3410000801 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             479 |              64 |            177 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498451.9910001755 |          5 |         3702498378 | 3702498451.9910001755 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             493 |              64 |            182 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered |  3702498458.767999649 |          5 |         3702498385 |  3702498458.767999649 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             397 |              67 |            167 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498462.2030000687 |          5 |         3702498389 | 3702498462.2030000687 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             483 |              66 |            182 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498463.2869997025 |          5 |         3702498390 | 3702498463.2869997025 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             467 |              67 |            189 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498464.4169998169 |          5 |         3702498391 | 3702498464.4169998169 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             390 |              65 |            195 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498465.5460000038 |          5 |         3702498392 | 3702498465.5460000038 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             469 |              68 |            208 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498466.6760001183 |          5 |         3702498393 | 3702498466.6760001183 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             409 |              65 |            203 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498467.8059997559 |          5 |         3702498394 | 3702498467.8059997559 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             427 |              64 |            232 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702412800 | telemetered | 3702498474.5840001106 |          5 |         3702498401 | 3702498474.5840001106 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             437 |              63 |            231 |        null

(35 rows)
